content/python/concepts/sets/terms/discard/discard.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.discard()'
+Description: 'Removes a passing element from the set if it's present.'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.discard()` method removes the specified item from the set of objects. 
+
+## Syntax
+
+The syntax is the following:
+
+```
+set_name.discard(value)
+```
+- This method takes exactly one argument;
```

Comment:
This line can be removed, instead of this line describe the parameters in the following format:
- `set`: Parameter description
- `value`: Parameter description